nix/nixpkgs.nix

```
@@ -7,6 +7,8 @@ let
     niv = (import sources.niv {}).niv;
     gitignore = import sources.gitignore { inherit (nixpkgs) lib; };
 
+    verilog-ethernet = import ./verilog-ethernet.nix { };
```

I guess it doesn't _really_ matter, but

```suggestion
    verilog-ethernet = import ./verilog-ethernet.nix { nixpkgs };
```

?